scripts/scil_filter_tractogram_anatomically.py

```diff
@@ -246,28 +323,42 @@ def main():
         ctx_mask == int(1))] = 1
 
     # Filter tractogram
-    loop_length_CSF_WM_sft , _ = filter_grid_roi(loop_length_CSF_sft,
+    loop_length_csf_wm_sft , _ = filter_grid_roi(loop_length_csf_sft,
         freesurfer_mask, 'both_ends', False)
 
     # Streamline count after final filtering
-    o_dict['streamline_count_after_filtering_length_loops_CSF_WM'] =\
-        len(loop_length_CSF_WM_sft.streamlines)
+    o_dict['streamline_count_after_filtering_length_loops_csf_wm'] =\
+        len(loop_length_csf_wm_sft.streamlines)
+
+    if args.save_volumes:
+        new_path = os.path.join(out_path, 'end_in_atlas')
+        if not os.path.exists(new_path):
+            os.mkdir(new_path)
+        nib.save(nib.Nifti1Image(freesurfer_mask, img_wmparc.affine,
+                img_wmparc.header), os.path.join(new_path,
+                    'atlas_bin' + '.nii.gz'))
+
+    if args.save_outliers:
+        outlier = compute_outliers(loop_length_csf_sft, loop_length_csf_wm_sft)
+        o_dict['streamline_count_discarded_when_filtering_wm_endings'] =\
+            len(outlier.streamlines)
+        save_outliers(out_path, 'end_in_atlas',
+            'length_no_loops_no_end_csf_end_in_atlas', in_tracto_name, ext,
```

Comment:
PEP 8: continuation line under-indented for visual indent

Reply:
Done in dbd0b2a.